src/main/java/com/communityratesgames/domain/Game.java

```diff
@@ -1,42 +1,38 @@
 package com.communityratesgames.domain;
 
 import javax.persistence.*;
+import java.io.Serializable;
 import java.sql.Timestamp;
 import java.util.List;
-import java.util.ArrayList;
-
-import com.communityratesgames.model.GameModel;
-import com.communityratesgames.model.PlatformModel;
 
 @Entity
 @Table(name = "game_entity")
-public class Game {
+public class Game implements Serializable {
```

Comment:
We should do that with all the classes. `Serializable` doesn't even have any definition requirements.
Maybe starting an issue about it is a good idea.